examples/data-solutions/data-platform-foundations/variables.tf

```diff
@@ -33,6 +33,17 @@ variable "composer_config" {
   }
 }
 
+variable "data_catalog_tags" {
+  description = "List of Data Catalog Policy tags to be created with optional IAM binging configuration in {tag => {ROLE => [MEMBERS]}} format."
+  type        = map(map(list(string)))
+  nullable    = false
+  default = {
+    "3_Confidential" = null
+    "2_Private"      = null
+    "1_Sensitive"    = null
+  }
+}
+
```

Comment:
You can set `nullable = false`, `default = {}` and then ignore the `count` in the module.

Reply:
good point. Done.